### PR TITLE
Update firebase analytics to 7.1.0 version

### LINKED ISSUE #15: Update to Firebase 7.0.1

### PR DESCRIPTION
**Pull request by ekharkunov**
Update firebase analytics to 7.1.0 version. Remove deprecated SetMinimumSessionDuration. Add opportunity to see events in Debug view for ios.

### CODE REVIEW

File: firebase/manifests/android/build.gradle [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 dependencies {
-    compile 'com.google.firebase:firebase-analytics:17.5.0'
-    compile 'com.google.android.gms:play-services-base:17.4.0'
+    implementation platform('com.google.firebase:firebase-bom:26.5.0')
```
Comment (britzl) [inline firebase/manifests/android/build.gradle:3]: Are you sure this is correct? I'm getting a crash with the example packaged with the extension:

```
firebase  E  Unable to check Google Play services availablity as the com.google.android.gms.common.Google
			 ApiAvailability class is not present in this application.
```

The dependency map shows this:

https://github.com/firebase/firebase-cpp-sdk/blob/a7e77482bea5ac5dfa646bd4a3bfc8a608bbb786/Android/firebase_dependencies.gradle#L23-L24

```
  'analytics' : ['com.google.firebase:firebase-analytics:18.0.2',
                 'com.google.android.gms:play-services-base:17.6.0'],
```

**Review by britzl — CHANGES_REQUESTED** [review]

**Comment by britzl** [comment]
We have probably not done this in the most recent library updates but we should also update the firebase/manifest/android/firebase-analytics.pro ProGuard file. There is a new one in the CPP SDK named analytics.pro.

File: firebase/manifests/android/build.gradle (hunk 1 above)
Comment (ekharkunov) [reply firebase/manifests/android/build.gradle:3]: Yes, you are right. I missed nessessary dependecy.

**Comment by ekharkunov** [comment]
Updated due to feedback.

**Review by britzl — APPROVED** [review]

**Pull request merged**